examples/seismic/model.py

```diff
@@ -347,6 +375,7 @@ def __init__(self, origin, spacing, shape, space_order, vp, nbpml=20,
 
         if epsilon is not None:
             if isinstance(epsilon, np.ndarray):
+                self._physical_parameters += ('epsilon',)
```

Comment:
No! This might create issues if we instantiate multiple, different Models. For the way ``_physical_parameters`` is used, we shouldn't let objects to modify this class attribute

Reply:
I don't follow. This is the initialization of a model, and this attached to this model its physical representation. I don't see what issue it could cause.

Reply:
It shouldn't which is why I originaly wrote it the other way. `_physical_parameters` is a class property that define what type of model you are working with and should not be shared between two models.

Reply:
ok, now sorted out